scripts/loggeranalyzer.py

```diff
@@ -85,37 +60,11 @@ class ErrorEnum(Enum):
     XML_PARSER_ERROR = "xml-parser-error"
     BUNDLE_NOT_FOUND = "bundle-not-found"
     ISSN_NOT_FOUND = "issn-not-found"
-    PACKAGE_NOT_FOUND = "package-not-found"
-
-
-PACK_FROM_SITE_PARSERS_PARAMS = [
-    {
-        "regex": ASSET_NOT_FOUND_REGEX,
-        "error": ErrorEnum.RESOURCE_NOT_FOUND,
-        "group": "renditions",
-    },
-    {
-        "regex": RENDITION_NOT_FOUND_REGEX,
-        "error": ErrorEnum.RESOURCE_NOT_FOUND,
-        "group": "renditions",
-    },
-    {"regex": XML_NOT_UPDATED_REGEX, "error": ErrorEnum.NOT_UPDATED},
-    {"regex": XML_NOT_FOUND_REGEX, "error": ErrorEnum.XML_NOT_FOUND},
-    {"regex": XML_MISSING_METADATA_REGEX, "error": ErrorEnum.MISSING_METADATA},
-]
-
-
-IMPORT_PARSERS_PARAMS = [
-    {"regex": MANIFEST_NOT_FOUND, "error": ErrorEnum.MISSING_MANIFEST},
-    {"regex": XML_NOT_INTO_PACKAGE, "error": ErrorEnum.XML_NOT_FOUND},
-    {"regex": XML_PARSER_ERROR, "error": ErrorEnum.XML_PARSER_ERROR},
-    {"regex": BUNDLE_NOT_FOUND, "error": ErrorEnum.BUNDLE_NOT_FOUND},
-    {"regex": ISSN_NOT_FOUND, "error": ErrorEnum.ISSN_NOT_FOUND},
-    {"regex": PACKAGE_NOT_FOUND, "error": ErrorEnum.PACKAGE_NOT_FOUND},
-]
+    PACKAGE_NOT_IMPORT = "package-not-import"
```

Comment:
Melhor: `PACKAGE_NOT_IMPORTED = "package-not-imported"`

Reply:
👍